sycl/test-e2e/bindless_images/dx12_interop/read_write_unsampled_semaphore.cpp

```diff
@@ -4,7 +4,8 @@
 // XFAIL: run-mode
 // XFAIL-TRACKER: https://github.com/intel/llvm/issues/15851
 
-// RUN: %{build} -l d3d12 -l dxgi -l dxguid -o %t.out
+// DEFINE: %{link-flags}=%if cl_options %{ /clang:-ld3d12 /clang:-ldxgi /clang:-ldxguid %} %else %{ -ld3d12 -ldxgi -ldxguid %}
```

Comment:
Could we move these to the config file as another substitution, e.g. `%link-directx`?

Reply:
Yeah that's a good idea, although right now there's only two tests linking directx, so maybe this is something to do when more directx tests are added.